samples/matter/common/src/Kconfig

```diff
@@ -83,6 +83,13 @@ config NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_MAX_TRIGGERS_DELEGATES
 	  Defines the maximum number for the TestEventTriggerDelegate implementations
 	  to be registered in the nRF test event triggers class.
 
+config NCS_SAMPLE_MATTER_TEST_SHELL
+	bool "Test - specific shell commands support for Matter samples"
+	depends on CHIP_LIB_SHELL
+	default y
```

Comment:
Do we want to enable it by default? I thought that it was going to be used mostly in CI.

Reply:
Fair point, maybe we should add it only in selected configuration in sample.yaml

Reply:
That was a request from QA  to enable it across all of our samples. It still depends on `CHIP_LIB_SHELL` so I guess it can be enabled by default

Reply:
ok, I'll discuss with QA which samples are most important for them

Reply:
discussed with @HubertHugo and decided on new configuration under template sample